src/pow.cpp

```diff
@@ -14,18 +14,14 @@
 #include "uint256.h"
 #include "util.h"
 
+#define NUM_OF_BLOCKS_DIFF_ADJ 10
```

Comment:
I'm afraid 10 blocks is not enough to prevent volatile difficulty fluctuation. But it can be discussed